dusty/Cargo.toml

```diff
@@ -5,10 +5,11 @@ authors = ["Benjamin Naecker <ben@oxidecomputer.com>"]
 edition = "2018"
 description = "Tool to inspect USDT probe records in object files"
 license = "Apache-2.0"
+rust-version = "1.70.0"
```

Comment:
Just explicitly added this given that there are multiple clap deps that require 1.70.0 as a minimum.